src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -217,7 +217,7 @@ internal PackageSourceMoniker SelectedSource
 
         public INuGetExperimentationService NuGetExperimentationService { get; private set; }
 
-        public bool ShowVulnerabilities => _topPanel.CheckBoxVulnerabilities.IsChecked == true;
+        public bool IsVulnerableFilteringApplied => _topPanel.CheckBoxVulnerabilities.IsChecked == true;
```

Comment:
I believe I saw a comment from you earlier that said that CheckBoxVulnerabilities can be null. If that's the case, we can't dereference it to check IsChecked without guarding against null.